types/block.go

```diff
@@ -1243,6 +1243,34 @@ type Message struct {
 	Data []byte
 }
 
+var (
+	MessageEmpty  = Message{}
+	MessagesEmpty = Messages{}
```

Comment:
I think these vars won't be used outside of this package? If so, they should be unexported.